modules/tracing/mule-tracer-configuration-api/src/main/java/org/mule/runtime/tracing/level/api/config/TracingLevelConfiguration.java

```diff
@@ -16,10 +16,15 @@
 public interface TracingLevelConfiguration {
 
   /**
-   * @return the default tracing level, MONITORING, if no other tracing level is specified from a configuration.
+   * @return the default tracing level, if no other tracing level is specified from a configuration.
```

Comment:
Lets put "@return The configured tracing level." 

The reason for this is that the default is also a configuration. Default configs must belong to the implementations, not the API.